.github/workflows/build_test.yml

```diff
@@ -26,6 +28,8 @@ jobs:
         override: true
     - name: Build relay node and parachain
       run: |
+        cargo update -p pallet-account-linker
```

Comment:
So we always test with latest pallet
And the trigger is from where? Intergration test here or in pallet